base/applications/mspaint/precomp.h

```diff
@@ -21,6 +21,9 @@
 #include <shellapi.h>
 #include <htmlhelp.h>
 
+#define NDEBUG
+#include <reactos/debug.h>
```

Comment:
`#include <debug.h>` should be sufficient? or would this break compilation? (if so, then just keep using `reactos/debug.h`)

Reply:
Used `<debug.h>`.